landing_page/static/landing_page/js/index.js

```diff
@@ -38,14 +38,25 @@ async function searchMovies(event) {
         .then((data) => {
             moviesContainer = document.getElementById("searched_movies_container")
             moviesContainer.innerHTML = '';
-            data.movies.forEach(movie => {
+            searchbar = document.getElementById("searchbar")
+            searchbar.style.borderRadius = "12px 12px 0 0"
+
+            data.movies.forEach((movie, index) => {
```

Comment:
I think it could be helpful if the poster and some info (runtime & release year for example. The MPA rating would also be nice to add but I don't have that in the db but I'll look into also scraping it when I populate the IMDb ID's) was also present by the title.

Reply:
This is added